modules/nf-core/paragraph/idxdepth/meta.yml

```diff
@@ -0,0 +1,60 @@
+name: "paragraph_idxdepth"
+description: Determines the depth in a BAM/CRAM file
+keywords:
+  - bam
+  - cram
+  - depth
+  - paragraph
+tools:
+  - "paragraph":
+      description: "Graph realignment tools for structural variants"
+      homepage: "https://github.com/Illumina/paragraph"
+      documentation: "https://github.com/Illumina/paragraph"
+      tool_dev_url: "https://github.com/Illumina/paragraph"
+      doi: "https://doi.org/10.1101/635011"
+      licence: "['Apache License 2.0']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - input:
+      type: file
+      description: BAM/CRAM file
+      pattern: "*.{bam,cram}"
+  - input_index:
+      type: file
+      description: Index of the BAM/CRAM file
+      pattern: "*.{bai,crai}"
+  - fasta:
+      type: file
+      description: Reference genome FASTA
+      pattern: "*.{fasta,fna,fa}"
+  - fasta:
+      type: file
+      description: Index of the reference genome FASTA
+      pattern: "*.fai"
```

Comment:
To bring up a discussion we had with my subworkflow: Wouldn't it make sense to have these two as one argument?

Reply:
To be honest, I like that too, but it's done like this in most modules. That's why I'm doing it like this here. But this can be discussed for sure

Reply:
OK.